budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/web/pages/budgets/overview/filter/BudgetTagFilterPanel.java

```diff
@@ -35,6 +36,8 @@ private Button createResetButton(String id, final IModel<List<String>> tagsModel
             @Override
             public void onSubmit() {
                 getFilter().getSelectedTags().clear();
+                BudgeteerSession.get().setBudgetRemainingFilter(0L);
```

Comment:
Is this the right place to set this value? The tag filter panel should not be concerned with the budget remaining filter.

Also, it do not understand why the response page has to be set here. Is this required by the other changes?

Reply:
Nope, this wasn't very pretty, fixed it as well ;)